inc/admin/namespace.php

```diff
@@ -73,16 +73,10 @@ function get_config() {
 		return [];
 	}
 
-	$settings['sp'] = [
-		'entityId'                 => $sp_home_url,
-		'assertionConsumerService' => [
-			'url' => $sp_base_url . 'verify',
-		],
-		'singleLogoutService'      => [
-			'url' => $sp_base_url . 'sls',
-		],
-		'NameIDFormat'             => 'urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress',
-	];
+	$settings['sp']['entityId']                 = $sp_home_url;
```

Comment:
This doesn't guarantee `$settings['sp']` is set any more.
```suggestion
	if ( ! isset( $settings['sp'] ) {
		$settings['sp'] = [];
	}

	$settings['sp']['entityId']                 = $sp_home_url;
```